docs/ext/commands/slash_commands.rst

```diff
@@ -200,6 +200,56 @@ The type of the option is determined by the range bounds, with the option being
         [tool.mypy]
         plugins = "disnake.ext.mypy_plugin"
 
+.. _string_lengths:
+
+String Lengths
+++++++++++++++
+
+:class:`str` parameters support minimum and maximum allowed value lengths.
+values using the ``min_length`` and ``max_length`` parameters on :func:`Param <ext.commands.Param>`.
+For instance, you could restrict an option to only accept a single character:
+
+.. code-block:: python3
+
+    @bot.slash_command()
+    async def charinfo(
+        inter: disnake.CommandInteraction,
+        character: str = commands.Param(max_length=1),
+    ):
+        ...
+
+Or restrict a tag command to limit tag names to 20 characters:
+
+.. code-block:: python3
+
+    @bot.slash_command()
+    async def tags(
+        inter: disnake.CommandInteraction,
+        tag: str = commands.Param(max_length=20)
+    ):
+        ...
+
+Instead of using :func:`Param <ext.commands.Param>`, you can also use a :class:`~ext.commands.String` annotation.
+The lengths bounds are both inclusive; using ``...`` as a bound indicates that this end of the string length is unbounded.
```

Comment:
```suggestion
The length bounds are both inclusive; using ``...`` as a bound indicates that this end of the string length is unbounded.
```

Additionally, is there a difference between `String[..., 10]` and `String[0, 10]`?

Reply:
The difference between 0 and ... for the minimum is basically nil, but I think its worth being able to not provide the minimum value anyways.

Reply:
https://github.com/DisnakeDev/disnake/pull/593/commits/9995b127debdef723982dbcd6d86748f931cdc13